test/test_cabinet.py

```diff
@@ -65,6 +65,25 @@ def test_read_create_pickle(self):
         result = cabinets.read(f'{protocol}://{filename}')
         self.assertDictEqual(data, result)
 
+    def test_read_plain_text(self):
+        protocol = 'file'
+        filename = os.path.join(self.fixture_path, 'sample.txt')
+        data = cabinets.read(f'{protocol}://{filename}')
+        expected = "I am sample text!\nThis file has more than one line.\n" \
```

Comment:
May want to add some more special cases to this test, e.g. unicode characters?

Reply:
Yeah, I just added some non-ascii characters, should be good enough.